src/main/java/com/google/api/gax/core/PropertiesProvider.java

```diff
@@ -56,14 +55,33 @@ public static String getGrpcVersion() {
     return grpcVersion != null ? grpcVersion : DEFAULT_VERSION;
   }
 
+  /**
+   * Utility method of retrieving the property value of the given key from a property file in the
+   * package.
+   *
+   * @param loadedClass The class used to get the resource path
+   * @param propertyFilePath The relative file path to the property file
+   * @param key Key string of the property
+   */
+  public static String loadProperty(Class<?> loadedClass, String propertyFilePath, String key) {
+    try {
+      Properties properties = new Properties();
+      properties.load(loadedClass.getResourceAsStream(propertyFilePath));
```

Comment:
Fair enough, but if the plan is to use this function in `getGapicVersion()` (https://github.com/googleapis/toolkit/pull/1134/files/bb5590c04ec747fddab5f7d351bc945c674e20f7#diff-b72b530711aeae89dad3e16aa67fe31aR113) then we will be loading from file every time.